en/docs/references/ballerina-expression-syntax.md

```diff
@@ -37,6 +37,26 @@ To initialize a variable of type `float` in Choreo, you can use one of the follo
   +950.930
   ```
 
+### decimal
+
+The `decimal` type supports integer values with up to 34 decimal points and supports the same arithmetic operators as `int`. When entering literals, you can use `d` as the suffix.
+
+This type does not support `infinity`, `NaN`, or negative zero.
```

Comment:
Why are we saying negative zero is not supported?

Reply:
Spec says "NaN, infinities and **subnormals** are not supported".

Reply:
```suggestion
This type does not support `infinity`, `NaN`, or subnormals.
```